app/controllers/DownloadRequestSuccessController.scala

```diff
@@ -40,7 +40,7 @@ class DownloadRequestSuccessController @Inject() (
 
   def onPageLoad: Action[AnyContent] = (identify andThen profileAuth andThen getData andThen requireData).async {
     implicit request =>
-      downloadDataConnector.getEmail(request.eori).map { email =>
+      downloadDataConnector.getEmail(request.eori).map { case Some(email) =>
```

Comment:
this makes some warnings so we need to provide a case _ with a sensible error message

Reply:
Done